src/main/js/components/subcomponents/DiningSession.js

```diff
@@ -23,4 +35,109 @@ export class CustomerDiningSessionSelect extends React.Component {
             </select>
         );
     }
+}
+
+export class StaffDiningSessionPage extends React.Component{
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            show : false,
+            orders : []
+        }
+    }
+
+    render(){
+        const sessions = this.props.diningSessions.map(session =>
+            <StaffDiningSession
+                key={session.entity._links.self.href}
+                diningSession={session}
+                diningSessionAttributes={this.props.diningSessionAttributes}
+                history = {this.props.history}
+                onCreate={this.onCreate}
+                onUpdate={this.onUpdate}
+                onDelete={this.onDelete}
+                onNavigate={this.onNavigate}
+            />)
+
+        //TODO review how to get number of diningSessions. Test whether this.props.diningSessions.size returns a number or undefined
+        return  (
+            <div>
+                <title>All Orders</title>
+                {sessions}
+                
+            </div>
+        )
+    }
+
+    handleClose(){
+        //load orders
+        // const orders;
+    }
+
+}
+
+export class StaffDiningSession extends React.Component{
+
+    constructor(props){
+        super(props);
+        this.handleListOrders = this.handleListOrders.bind(this);
+    }
+
+    requestOrders(){
```

Comment:
This function seems to be duplicated from `Order.js` **DiningSessionOrders**
Nor is there a binding for this function within it's class (_StaffDiningSession_)